windows/server_windows.py

```diff
@@ -0,0 +1,8 @@
+"""Server initialization code for Windows."""
+# SPDX-FileCopyrightText: 2021 Timeseer.AI
+#
+# SPDX-License-Identifier: Apache-2.0
+from kukur.cli import run
+
+if __name__ == '__main__':
+    run()
```

Comment:
Since this file does not do anything else than call `run()`, is it needed? I would assume that we can use the normal CLI.